op-program/host/prefetcher/prefetcher_test.go

```diff
@@ -153,13 +159,94 @@ func TestFetchL1Receipts(t *testing.T) {
 	})
 }
 
+// Globally initialize a kzgCtx for blob tests.
+var kzgCtx, _ = gokzg4844.NewContext4096Secure()
+
+func deterministicRandomness(seed int64) [32]byte {
+	// Converts an int64 to a byte slice
+	buf := new(bytes.Buffer)
+	err := binary.Write(buf, binary.BigEndian, seed)
+	if err != nil {
+		log.Error("Failed to write int64 to bytes buffer: %v", err)
+	}
+	bytes := buf.Bytes()
+
+	return sha256.Sum256(bytes)
+}
+
+// Returns a serialized random field element in big-endian
+func GetRandFieldElement(seed int64) [32]byte {
+	bytes := deterministicRandomness(seed)
+	var r fr.Element
+	r.SetBytes(bytes[:])
```

Comment:
Do the docs say if this applies modulo to the input bytes, or is it undefined behavior?